boards/qualcomm/qcc744m_evk/qcc744m_evk.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2025 MASSDRIVER EI (massdriver.space)
+#
+# SPDX-License-Identifier: Apache-2.0
+
+identifier: qcc744m_evk
+name: Qualcomm QCC744M Evaluation Board
+type: mcu
+arch: riscv
+ram: 480
+toolchain:
+  - zephyr
+testing:
+  ignore_tags:
+    - net
+    - bluetooth
+supported:
+  - gpio
+  - pinctrl
+  - uart
```

Comment:
If wanting to also list the GPIO button as an on-board supported feature:

```
  - input
```

![Image](https://github.com/user-attachments/assets/1bcf3406-a8fe-4623-a52d-41d5e99d3c20)